core/serialization/download_json_serializer.py

```diff
@@ -269,11 +259,10 @@ def __init__(self, outcome_data: entities.OutcomeData):
 
     def to_json(self):
         return {
-            "ID": str(self.outcome_data.id),
             "SubmissionID": self.outcome_data.submission.submission_id,
             "ProjectID": self.outcome_data.project.project_id,
-            "FinancialPeriodStart": str(self.outcome_data.start_date),
-            "FinancialPeriodEnd": str(self.outcome_data.end_date),
+            "StartDate": str(self.outcome_data.start_date),
```

Comment:
I'm just checking that these are definitely now right, as I have a version that I thought was the most up-to-date, but differs from the trello ticket.